uk_bin_collection/uk_bin_collection/councils/BromleyBoroughCouncil.py

```
@@ -8,7 +8,7 @@
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.support.ui import WebDriverWait
 
-from uk_bin_collection.uk_bin_collection.common import create_webdriver
+from uk_bin_collection.uk_bin_collection.common import *
```

Ruff (0.14.14)

[error] `from uk_bin_collection.uk_bin_collection.common import *` used; unable to detect undefined names

(F403)